openedx/features/course_experience/templates/course_experience/course-home-fragment.html

```diff
@@ -200,4 +203,22 @@ <h3 class="hd-6 section-title">${_("Course Handouts")}</h3>
       linkCategory: "personalized_learner_schedules"
     });
 
-</%static:require_module_async>
+    TrackECommerceEvents.trackUpsellClick(upgradeDateLink, 'course_home_dates', {
+      pageName: "course_home",
+      linkType: "link",
+      linkCategory: "(none)"
+    });
+
+    TrackECommerceEvents.trackUpsellClick(GreenUpgradeLink, 'course_home_green', {
+      pageName: "course_home",
+      linkType: "button",
+      linkCategory: "green_upgrade"
+    });
+
+    TrackECommerceEvents.trackUpsellClick(courseToolsUpgradeLink, 'course_home_course_tools', {
+      pageName: "course_home",
+      linkType: "link",
+      linkCategory: "(none)"
```

Comment:
what does link category none mean?

Reply:
(see description of categories in the last comment, let me know if you have any other questions about it!)